tests/unit/wmg_processing/test_load_corpus.py

```diff
@@ -32,25 +32,26 @@ def fixture_file_path(relative_filename):
         parent_dir_of_curr_file = "/".join(os.path.dirname(__file__).split("/")[:-1])
         return os.path.abspath(os.path.join(parent_dir_of_curr_file, relative_filename))
 
+    @staticmethod
+    def create_anndata_test_object(path, dataset_name, num_genes=1000, num_cells=5000):
```

Comment:
I suppose this could also live in `tests.unit.backend.wmg.fixtures.test_anndata_object`